.github/workflows/ci.yml

```diff
@@ -728,7 +728,13 @@ jobs:
         python -m pip install --upgrade pip
         python -m pip install pytest
         python -m pip install pytest-xdist
+        python -m pip install numba
         echo ::endgroup::
+
+        echo ::group::Run complete test suite
+        python -m pytest
```

Comment:
We need exit 1 or to remove `continue-on-error: true` and do `|| true` on the expected to fail lines...

Reply:
done